VimeoUpload/Extensions/URL+MIMEType.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  URL+MIMEType.swift
+//  VimeoUpload
+//
+//  Created by Lehrer, Nicole on 2/4/19.
+//
+
+import MobileCoreServices
+
+public extension URL
+{
+    /// A helper for determining a file's MIMEType
+    ///
+    /// - Returns: MIMEType as String
+    /// - Throws: throws an error if the MIMEType cannot be determined
+    public func MIMEType() throws -> String {
```

Comment:
Wouldn't `mimeType()` be preferable? IIRC, we're suppose to use lowercase for the first word, even if it's an acronym, `urlAsset` or `httpBody` come to mind.

Reply:
Yes